sdk/schemaregistry/Azure.Data.SchemaRegistry/api/Azure.Data.SchemaRegistry.netstandard2.0.cs

```diff
@@ -7,6 +7,8 @@ namespace Azure.Data.SchemaRegistry
         private readonly int _dummyPrimitive;
         public SchemaFormat(string value) { throw null; }
         public static Azure.Data.SchemaRegistry.SchemaFormat Avro { get { throw null; } }
+        public static Azure.Data.SchemaRegistry.SchemaFormat Custom { get { throw null; } }
```

Comment:
I think we need some tests for Custom. What is the content type for custom?

Reply:
Added the stubs for them but it seems like there isn't a true consensus on what the API will look like for custom, so I will add those in once the new swagger is in